compatibility.bs

```diff
@@ -995,32 +998,34 @@ On Firefox for Android, "<code>&lt;<a>platform</a>&gt;; &lt;<a for=firefox>devic
 <h4 id="ua-string-pattern-safari">Safari User-Agent pattern</h4>
 
 "<code>Mozilla/5.0 (&lt;<a>safariPlatform</a>&gt;) AppleWebKit/605.1.15 (KHTML, like Gecko)
-Version/&lt;<a>safariVersion</a>&gt;&lt;<a for="/">deviceCompat</a>&gt;Safari/605.1.15</code>"
+Version/&lt;<a>safariVersion</a>&gt;&lt;<a for="/">deviceCompat</a>&gt;Safari/&lt;<a>webkitVersion</a>&gt;</code>"
 
 <div class="example" id="safari-ua-examples">
   <!-- https://github.com/WebKit/WebKit/blob/8427f75c2c505bbeac2898839e57c027f0bfccd8/Source/WebCore/platform/mac/UserAgentMac.mm -->
   <strong>Desktop</strong>: "<code>Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15
-  (KHTML, like Gecko) Version/<mark>15.2</mark> Safari/605.1.15</code>"
+  (KHTML, like Gecko) Version/<mark>15.6</mark> Safari/<mark>605.1.15</mark></code>"
 
   <!-- https://github.com/WebKit/WebKit/blob/0935c1bc383b253a9b54232939d6e68eb60f38fe/Source/WebCore/platform/ios/UserAgentIOS.mm -->
   <strong>Mobile</strong>: "<code>Mozilla/5.0 (<mark>iPhone</mark>; CPU <mark>iPhone OS</mark>
-  <mark>15_3</mark> like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko)
-  Version/<mark>15.2</mark> <mark>Mobile/15E148</mark> Safari/605.1.15"
+  <mark>15_6</mark> like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko)
+  Version/<mark>15.6</mark> <mark>Mobile/15E148</mark> Safari/<mark>604.1</mark>"
 </code>
 </div>
 
 <h4 id="ua-string-tokens-safari">Safari-specific tokens</h4>
 
-<code>&lt;<dfn>safariVersion</dfn>&gt;</code> decomposes to the following:
+<code>&lt;<dfn>safariVersion</dfn>&gt;</code> <a>decomposes</a> to the following:
 
-<code>&lt;<a>majorVersion</a>&gt;.&lt;<a>minorVersion</a>&gt;</code>, where
+"<code>&lt;<a>majorVersion</a>&gt;.&lt;<a>minorVersion</a>&gt;</code>", where
 <code>&lt;<a>minorVersion</a>&gt;</code> is a single digit.
 
-<code>&lt;<dfn>safariPlatform</dfn>&gt;</code> decomposes to the following:
+<code>&lt;<dfn>safariPlatform</dfn>&gt;</code> <a>decomposes</a> to the following:
+
+On desktop, the <a>constant</a> "<code>Macintosh; Intel Mac OS X 10_15_7</code>".
 
-On desktop, the string "<code>Macintosh; Intel Mac OS X 10_15_7</code>".
+On larger iPad form factors, the <a>constant</a> "<code>Macintosh; Intel Mac OS X 10_15_6</code>".
```

Comment:
This recently changed (https://github.com/WebKit/WebKit/commit/2cc95bc67eb6a3ef70de894d03cc1d446c7530eb).

Reply:
Thanks @gsnedders 
Could you change this @miketaylr

Reply:
That's great - thanks @gsnedders